client/components/Dashboard.jsx

```diff
@@ -1,10 +1,26 @@
 import React from 'react'
 
+import ProgressModule from './ProgressModule'
 
 export default function Dashboard() {
+  const placeHolderProps = {
+    tally: 20
```

Comment:
could this be changed to percentage? Tally doesn't quite make sense in this context.